backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -50,4 +54,27 @@ public UserModel getUserById(
 		return userModelAssembler.toModel(user);
 	}
 
+
+    @PostMapping({ "/{userId}/email-validations"})
+    @Operation(summary = "Verify the status of a confirmation code.", operationId="verify-confirmation-code")
+    public ResponseEntity<?>  verifyConfirmationCodeStatus(
+            @NotBlank(message = "userId must not be null or blank")
+            @Parameter(description = "The ID of the user.", required = true)
+            @PathVariable String userId,
+            @NotBlank(message = "code must not be null or blank")
+            @Parameter(description = "The confirmation code.", required = true)
+            @RequestParam String code
+        ){
+			final var user = userService.getUserById(userId)
+				.orElseThrow(() -> new ResourceNotFoundException("No user with id=[%s] was found".formatted(userId)));
+			Assert.hasText(code, "id is required; it must not be null or blank");
+			if (userService.verifyConfirmationCode(code, user)) {
+				userService.setEmailValidated(userId);
+				return new ResponseEntity<>(HttpStatus.ACCEPTED);
+			}
+			return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
+		
+
```

Comment:
```suggestion
```

Reply:
Superfluous whitespace removed in commit 2cebd2d6f0685660fa512c66c4a5da16869070e1..

Reply:
Going to hijack this comment but there is also a mix of spaces and tabs for indentation. Turning on whitespace characters will help identifiy this:

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/5bea47bd-f195-4204-8bf1-6fe46ba92454)

Reply:
Whitespace has been cleaned up in commit 70d88ff21299e8b6f77785dd0b22d60b10f1a733. I think I got it all.